features/staging_commands.feature

```diff
@@ -59,7 +59,29 @@ Feature: Enable command behaviour when attached to an UA staging subscription
     @uses.config.machine_type.lxd.vm
     Scenario Outline: Attached enable of vm-based services in an ubuntu lxd vm
         Given a `<release>` machine with ubuntu-advantage-tools installed
-        When I attach `contract_token_staging` with sudo
+        When I create the file `/tmp/machine-token-overlay.json` with the following:
+        """
+        {
+            "machineTokenInfo": {
+                "contractInfo": {
+                    "resourceEntitlements": [
+                        {
+                            "type": "fips",
+                            "directives": {
+                                "additionalPackages": ["ubuntu-fips"]
+                            }
+                        }
+                    ]
+                }
+            }
+        }
+        """
```

Comment:
Why do we need to set this when talking to staging?
If I look at fips directives on a plain staging attach: 
```json
                  "additionalPackages" : [
                     "ubuntu-fips"
                  ],
```

Reply:
Shouldn't this be in an integration test tagged w/ `@uses.config.machine_type.aws.generic` ?

Reply:
You are right here. This is a mistake. Originally, I tried to use the same fips test for all machine types, but it did not work out and I forgot to remove those steps. I will update the code